### PR TITLE
Muzima 524

### PR DESCRIPTION
**Pull request by yoctopus**
Added the copyright information in every file

### CODE REVIEW

File: app/src/main/java/com/muzima/model/AvailableForm.java [hunk 1]
```diff
@@ -32,14 +33,7 @@ public void setDownloaded(boolean downloaded) {
...
-        if(tags == null){
-            return false;
-        }
-        for (Tag tag : tags) {
-            if(REGISTRATION.equalsIgnoreCase(tag.getName())){
-                return true;
-            }
-        }
-        return false;
+        return Constants.FORM_DISCRIMINATOR_REGISTRATION.equalsIgnoreCase(getDiscriminator()) ||
```
Comment (mssavai) [inline app/src/main/java/com/muzima/model/AvailableForm.java]: Hi @yoctopus , your commit includes changes already submitted in a separate pull request (MUZIMA-330). There will be a conflict when processing the other PR  if we merge this as it is. Could you revert non-copyright info changes? Thanks.

Comment (yoctopus) [inline app/src/main/java/com/muzima/model/AvailableForm.java]: Is is possible to cancel a pull request, let me try that.
